test/test_devices.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from absl.testing import absltest, parameterized
+import torch
+import torch_xla as xla
+import torch_xla.runtime as xr
+
+
+class TestDevices(parameterized.TestCase):
+
+  def setUpClass():
+    xr.set_device_type('CPU')
+    os.environ['CPU_NUM_DEVICES'] = '4'
```

Comment:
hmm, it is OK for now but shouldn't we also test it on GPU and TPU?

Reply:
This is sufficient IMO. We're really just testing the integration of this module with the runtime client, which has the same API regardless of the underlying device.

As we switch to using these functions by convention, they'll be exercised by almost every other test.